haystack/utils.py

```diff
@@ -79,3 +81,80 @@ def convert_labels_to_squad(labels_file: str):
 
     with open("labels_in_squad_format.json", "w+") as outfile:
         json.dump(labels_in_squad_format, outfile)
+
+def find_answer_start(answer: str, text: str):
+    """
+    get index of beginning of answer in text
+    :param text: string where 'answer' is to be searched in
+    :param answer: substring to be searched in 'text'
+    :return: list of indices of occurrences of answer
+    """
+    # escape all (
+    answer = r"\b{}\b".format(re.escape(answer))
+    return [m.start() for m in re.finditer(answer, text)]
```

Comment:
What happens for these cases: 
- answer not found in text?
- multiple answers found?
Do we ensure that we have in the end correct labels indexed even if those cases happen? Does the user notice if he he has such cases in his data?

Reply:
right now, multiple answers are treated as different 'answer's in the list of answers for a question in `qas`. 
I'll add a warning for no answer found and for multiple answers but handle the multiple answers as I have mentioned